src/debug/debug.proto

```diff
@@ -23,6 +23,7 @@ message Filter {
     bool pachd = 1;	
     pps_v2.Pipeline pipeline = 2;	
     Worker worker = 3;
+    bool database = 4;
```

Comment:
I think this maybe shouldn't be a filter yet.  Filter implies "collect one of these things", and we are intending it to be part of the pachd dump.  Maybe when we do the full database dump, we'll want to make that an option for the pachd dump?  (Annoyingly, that's going to be hard to do.  This proto should probably be `oneof filter { PachdDumpConfig pachd = 1; ... }` and not just a raw bool.  Sigh!)

Reply:
You make a good point, but having a filter just for database files does help when testing